packages/entity-server/src/index.ts

```diff
@@ -19,7 +20,7 @@ dotenv.config(); // Load the environment variables into process.env
 /**
  * Set up models for Authenticator
  */
-const models = new ModelRegistry(new TupaiaDatabase());
+const models = new ModelRegistry(new TupaiaDatabase()) as EntityServerModelRegistry;
```

Comment:
Unfortunately resorted to typecasting here...

I tried instantiating a class that extended `ModelRegistry` and only had the `entity` field declared however that ended up with `entity` being `undefined`... not exactly sure what went wrong there, perhaps worthy of more investigation...